ui/src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -34,14 +38,28 @@ interface StateProps {
   org: Organization
 }
 
-type Props = WithRouterProps & StateProps
+interface OwnProps extends WithRouterProps {
+  params: {templateName: string}
+}
+
+type Props = OwnProps & StateProps
 
 @ErrorHandling
-class CTI extends Component<Props> {
+class UnconnectedTemplatesIndex extends Component<Props> {
   state = {
     currentTemplate: '',
   }
 
+  public componentDidMount() {
+    if (this.props.params.templateName) {
+      this.setState({
+        currentTemplate: getGithubUrlFromTemplateName(
```

Comment:
is the input limited to github? I didn't' think it was

Reply:
i don't think it is, but right now for the mvp version 1, we're using https://github.com/influxdata/community-templates/, so that's what i'm targeting. yagni and what have you.

Reply:
see comment below